pandas/tests/test_base.py

```diff
@@ -35,10 +36,12 @@
 )
 from pandas.core.accessor import PandasDelegate
 from pandas.core.arrays import DatetimeArray, PandasArray, TimedeltaArray
-from pandas.core.base import NoNewAttributesMixin, PandasObject
+from pandas.core.base import IndexOpsMixin, NoNewAttributesMixin, PandasObject
 from pandas.core.indexes.datetimelike import DatetimeIndexOpsMixin
 import pandas.util.testing as tm
 
+index_or_series = [Index, Series]  # type: List[Type[IndexOpsMixin]]
```

Comment:
Are you not reusing the definition in `pandas._typing` here for a particular reason?

Reply:
Missed this one.